src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/TreeView/TreeViewPage.xaml.cs

```diff
@@ -0,0 +1,1067 @@
+				else
+				{
+					selectedNodes.Add(node0);
+					selectedNodes.Add(node2);
+				}
+			}
+		}
+
+		private void AddInheritedTreeViewNode_Click(object sender, RoutedEventArgs e)
+		{
+			if (_visualRoot != null)
+			{
+				var node = new TreeViewNode2() { Content = "Inherited from TreeViewNode" };
+				_visualRoot.Children.Add(node);
+			}
+		}
+
+		private void ClearNodes_Click(object sender, RoutedEventArgs e)
+		{
+			try
+			{
+				if (IsInContentMode())
+				{
+					TestTreeViewItemsSource.Clear();
+				}
+				else if (_visualRoot != null)
+				{
+					_visualRoot.Children.Clear();
+					_visualRoot = null;
+					TestTreeView.RootNodes.Clear();
+				}
+			}
+			catch (Exception ex)
+			{
+				ExceptionMessage.Text = ex.ToString();
+			}
+		}
+
+		private void AddRootNode_Click(object sender, RoutedEventArgs e)
+		{
+			try
+			{
+				if (IsInContentMode())
+				{
+					var newNode = new TreeViewItemSource() { Content = "Root" + TestTreeViewItemsSource.Count };
+					TestTreeViewItemsSource.Add(newNode);
+				}
+				else
+				{
+					var newNode = new TreeViewNode() { Content = "Root" + TestTreeView.RootNodes.Count };
+					TestTreeView.RootNodes.Add(newNode);
+					if (_visualRoot == null)
+					{
+						_visualRoot = newNode;
+					}
+				}
+			}
+			catch (Exception ex)
+			{
+				ExceptionMessage.Text = ex.ToString();
+			}
+		}
+
+		private void AddExtraNodes_Click(object sender, RoutedEventArgs e)
+		{
+			for (int i = 1; i <= 50; i++)
+			{
+				TestTreeView.RootNodes.Add(new TreeViewNode() { Content = "Node " + i });
+			}
+		}
+
+		private void SelectLastRootNode_Click(object sender, RoutedEventArgs e)
+		{
+			TestTreeView.SelectedNode = TestTreeView.RootNodes[TestTreeView.RootNodes.Count - 1];
+		}
+
+		private void TreeViewLateDataInitTestPage_Click(object sender, RoutedEventArgs e)
+		{
+			GetFrame().Navigate(typeof(TreeViewLateDataInitTest));
+		}
+
+		private void ToggleRoot0Selection_Click(object sender, RoutedEventArgs e)
+		{
+			TestTreeView.SelectedNode = TestTreeView.SelectedNode == null ? TestTreeView.RootNodes[0].Children[0] : null;
+			ContentModeTestTreeView.SelectedItem = ContentModeTestTreeView.SelectedItem == null ? TestTreeViewItemsSource[0].Children[0] : null;
+		}
+
+		private void TreeViewNodeInMarkupTestPage_Click(object sender, RoutedEventArgs e)
+		{
+			GetFrame().Navigate(typeof(TreeViewNodeInMarkupTestPage));
+		}
+
+		private void TreeViewUnrealizedChildrenTestPage_Click(object sender, RoutedEventArgs e)
+		{
+			GetFrame().Navigate(typeof(TreeViewUnrealizedChildrenTestPage));
+		}
+
+		private void ClearException_Click(object sender, RoutedEventArgs e)
+		{
+			ExceptionMessage.Text = string.Empty;
+		}
+
+		private async void ResetItemsSourceAsync_Click(object sender, RoutedEventArgs e)
+		{
+			// TODO: Uno Specific - Bindings.Update does not work correctly, setting the value directly instead.
+			ContentModeTestTreeView.ItemsSource = await PrepareItemsSourceAsync();
+			//this.Bindings.Update();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)